translate/convert/web2py2po.py

```diff
@@ -40,12 +42,14 @@ def convertunit(self, source_str, target_str):
         return pounit
 
     def convertstore(self, mydict):
-
         targetheader = self.mypofile.header()
         targetheader.addnote("extracted from web2py", "developer")
 
         for source_str in mydict.keys():
             target_str = mydict[source_str]
+            if sys.version_info[0] == 2:
```

Comment:
I would rather use `if six.PY2:` since that is what is more frequently used in our codebase. It might also help when stripping Python 2 support now that we are less than one year away from Python 2 not being supported anymore.